vertx-web-client/src/main/java/io/vertx/ext/web/client/impl/HttpRequestImpl.java

```diff
@@ -90,6 +98,7 @@ private HttpRequestImpl(HttpRequestImpl<T> other) {
     this.client = other.client;
     this.serverAddress = other.serverAddress;
     this.options = other.options;
+    this.proxyOptions = other.proxyOptions;
```

Comment:
proxy options should be copied here when not null

Reply:
Would other.proxyOptions != null ? other.proxyOptions.copy() : null suffice?

Reply:
ping

Reply:
This is already changed as requested